run_presubmit.sh

```diff
@@ -20,13 +20,13 @@ set -euo pipefail
 echo ========== Running unit tests.
 if [[ -z `which coverage` ]];then
```

Comment:
Just a general question: is it relatively fast to just install the tools required?  eg, if this were apt, I'd just install everything rather than check/install.

Not something to change in this PR, just a general question about pip.